code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/AlekseevSergeiTest.java

```diff
@@ -0,0 +1,78 @@
+package ru.unn.agile.ComplexNumber.model;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AlekseevSergeiTest {
+
+    @Test
+    public void areAddingZeroDoesntChangeNumber() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+        ComplexNumber zero = new ComplexNumber(0.0, 0.0);
+
+        ComplexNumber sum = z.add(zero);
+
+        assertTrue(sum.equals(z));
+    }
+
+    @Test
+    public void areMultiplicationByZeroChangeNumber() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+        ComplexNumber zero = new ComplexNumber(0.0, 0.0);
+
+        ComplexNumber mult = z.multiply(zero);
+
+        assertTrue(mult.equals(zero));
+    }
+
+    @Test
+    public void areDifferentNumbersEqual() {
+        ComplexNumber z1 = new ComplexNumber(1.1, 2.3);
+        ComplexNumber z2 = new ComplexNumber(5.8, 13.21);
+
+        assertFalse(z1.equals(z2));
+    }
+
+    private final double delta = 0.00000001;
+
+    @Test
+    public void canGetRealPart() {
+        ComplexNumber z = new ComplexNumber(1.1, 2.3);
+
+        assertEquals(1.1, z.getReal(), delta);
```

Comment:
You use magic numbers in assert, please, change it with something like that:

    public void canGetRealPart() {
        //Arrange
        ComplexNumber z = new ComplexNumber(1.1, 2.3);

        //Act
        var realPartComplex = z.getReal(); 
        var expectedRealPart = 1.1;

        //Assert
        assertEquals(expectedRealPart, realPartComplex, delta);